templates/registration/registration_email.html

```diff
@@ -1,5 +1,5 @@
 {% autoescape off %}
 Hi {{ user.username }},
 Please click on the link to confirm your registration,
-http://{{ domain }}{% url 'activate' uidb64=uid token=token %}
+http://localhost:8000{% url 'activate' uidb64=uid token=token %}
```

Comment:
Why hardcode the domain? In case this app will be hosted, the email will send the user to localhost:8000, which, on his machine, the address means nothing. It was fine before.

Reply:
For some reason, it did not work. {{ domain }} did not display anything